Makefile

```diff
@@ -155,7 +163,7 @@ controller-gen: ## Download controller-gen locally if necessary.
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 .PHONY: kustomize
 kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
+	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.2)
```

Comment:
nit: it's probably rewrite commands to use `kubectl apply -k xxx` instead of `kustomize build xxx | kubectl apply -f -`

Reply:
This makes sense, but let's leave it as is for now.